pkg/cloudprovider/aws/suite_test.go

```diff
@@ -51,8 +52,27 @@ var launchTemplateCache *cache.Cache
 var fakeEC2API *fake.EC2API
 var controller reconcile.Reconciler
 
+type singletonFS struct {
```

Comment:
consider making a fake.Filesystem like the fake.EC2API

Reply:
I was thinking of holding off until we see if there are more uses, lmk if that seems like a bad idea.

Reply:
Actually, I take that back - will do.

Reply:
Done